tools/typescript2fabel.js

```diff
@@ -240,9 +241,15 @@ function visitInterface(node) {
             case ts.SyntaxKind.PropertySignature:
                 ifc.properties.push(getProperty(node));
                 break;
+            case ts.SyntaxKind.PropertyDeclaration:
+                ifc.properties.push(getProperty(node));
+                break;
```

Comment:
What's the difference between Property Signature and Declaration? (Also for methods) I'm still puzzled by TypeScript SyntaxKind elements (they're so much more complicated than the symbols exposed by F# compiler services!). I wrote this script mostly by trial and error using astexplorer.net with typescript parser.